### PR TITLE
Add Datadog docs

### PR DESCRIPTION
**Pull request by simon-johansson**

### CODE REVIEW

File: docs/observability/metrics.md [hunk 1]
```diff
@@ -96,12 +96,24 @@ Then select **Grafana Cloud** in the settings menu and click on **Add Stack**.
...
 
 <img width="60%" src="/assets/docs/configstack.png" title="Select Grafana Stack"/>
 
 That's it! After your next deploy, Encore will start sending metrics data to your Grafana Cloud Stack.
 
 ### Datadog
 
-Coming soon! Reach out on [Slack](https://encore.dev/slack) if you are interested in learning more.
\ No newline at end of file
+To send metrics data to Datadog, you first need to Add a Datadog Account to your application.
```
Comment (marcuskohlberg) [inline docs/observability/metrics.md:107]:
```suggestion
To send metrics data to Datadog, you first need to add a Datadog Account to your application.
```

**Review by marcuskohlberg — APPROVED** [review]

File: docs/observability/metrics.md [hunk 2]
```diff
@@ -96,12 +96,24 @@ Then select **Grafana Cloud** in the settings menu and click on **Add Stack**.
...
 ### Datadog
 
-Coming soon! Reach out on [Slack](https://encore.dev/slack) if you are interested in learning more.
\ No newline at end of file
+To send metrics data to Datadog, you first need to add a Datadog Account to your application.
+
+Open your application on [app.encore.dev](https://app.encore.dev), and click on **Settings** in the main navigation.
+Then select **Datadog** in the settings menu and click on **Add Account**.
+
+<img width="60%" src="/assets/docs/datadogaccount.png" title="Add a Datadog account"/>
```
Comment (DomBlack) [inline docs/observability/metrics.md:112]: Are these images already in the repo or did you miss them on the PR?

**Review by DomBlack — APPROVED** [review]

File: docs/observability/metrics.md (hunk 2 above)
Comment (marcuskohlberg) [reply docs/observability/metrics.md:112]: They are on www already

Comment (DomBlack) [reply docs/observability/metrics.md:112]: Hmm, good point but I wonder if the docs should self host their images too, so people reading the doc's locally also have the images

**Pull request merged**